pkg/api/customization/project/project_actions.go

```diff
@@ -289,6 +290,15 @@ func (h *Handler) setPodSecurityPolicyTemplate(actionName string, action *types.
 		return fmt.Errorf("could not convert: %v", providedPSP)
 	}
 
+	if ok {
+		if _, err := h.PspTemplateLister.Get("", podSecurityPolicyTemplateName); err != nil {
```

Comment:
standard capitalization rule in golang is that abbreviations such as psp should be capitalized like PSP

Reply:
fixed.